.github/workflows/onRelease.yml

```diff
@@ -0,0 +1,18 @@
+# when a github release happens, publish an npm package,
+on:
+  release:
+    types: [released]
+  # support manual release
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: tag that needs to publish
+        type: string
+        required: true
+jobs:
+  npm:
+    uses: salesforcecli/github-workflows/.github/workflows/npmPublish.yml@main
+    with:
+      ctc: false
```

Comment:
Avoiding CTC intentionally?

Reply:
yup...we didn't do it in circle so I assume it's cool to not do it in GHA